src/Employer/Employer.Web/ViewModels/ApplicationReview/ApplicationReviewStatusConfirmationEditModel.cs

```diff
@@ -0,0 +1,9 @@
+using Esfa.Recruit.Shared.Web.ViewModels.ApplicationReview;
+
+namespace Esfa.Recruit.Employer.Web.ViewModels.ApplicationReview
+{
+    public class ApplicationReviewStatusConfirmationEditModel : ApplicationReviewEditModel, IApplicationStatusConfirmationEditViewModel
```

Comment:
Basing from ApplicationReviewEditModel is bit excessive. I would just add required properties to the confirmation vm and then I don't need the interface either.